src/graphql_sqlalchemy/objects.py

```diff
@@ -25,6 +27,8 @@ def get_fields() -> GraphQLFieldMap:
             graphql_type: GraphQLOutputType = get_graphql_type_from_column(column)
             if not column.nullable:
                 graphql_type = GraphQLNonNull(graphql_type)
+            if isinstance(column.type, (ARRAY, PGARRAY)):
+                graphql_type = GraphQLList(graphql_type)
```

Comment:
Would it make more sense to include this in `scalars/get_graphql_type_from_column` directly?

It would have the nice effect of changing the return type of that function to `GraphQLOutputType`

Reply:
I already tried doing that (was my first attempt) but somehow that function is probably called somewhere else too (I printed the column type and graphql type assigned and when I changed that function it was printed way more than here) and it ended up not working